src/stories/Filter/ListFilter/ListFilter.stories.tsx

```diff
@@ -41,7 +41,16 @@ const ListFilterTemplate: Story = (args: ListFilterArgs) => {
     return <FilterPage pageFilters={pageFilters} />;
 };
 
+ const ListFilterTemplateOneValue: Story = (args: ListFilterArgs) => {
```

Comment:
`ListFilterTemplateOneValue` can be removed. Creating a new template is not needed since it does the same thing as the first template. All you need to do is declare a second/new const and bind to the first template like:

```
export const ListFilterOneInitialValue = ListFilterTemplate.bind({});
```